src/DocumentationBrowserViewExtension/DocumentationBrowserViewModel.cs

```diff
@@ -245,7 +248,9 @@ private string LoadContentFromResources(string name)
             {
                 result = reader.ReadToEnd();
             }
-            return result;
+
+            // Clean up possible script tags from document
+            return Regex.Replace(result, SCRIPT_TAG_REGEX, "");
```

Comment:
I'm curious if you think logging any removals to the console would be good? I can see some developers getting really confused.

Reply:
I'm not sure I understand what you mean by logging to console, but thinking about it, maybe we could replace the tag by some content instead of "". Something like `SCRIPT TAG REMOVED` maybe?